lib/Elastica/Document.php

```diff
@@ -12,7 +12,8 @@
  */
 class Document extends AbstractUpdateAction
 {
-    const OP_TYPE_CREATE = Action::OP_TYPE_CREATE;
+    public const OP_TYPE_CREATE = Action::OP_TYPE_CREATE;
+    public const DEFAULT_TYPE = '_doc';
```

Comment:
Could `DEFAULT_TYPE` may be removed as it's not used?

Reply:
Removed!